### ISSUE TITLE
Pyright generates incomplete completion when overriding an asynchronous method

### ISSUE THREAD

**Issue by dob9601** [microsoft/pyright#1765]
**Describe the bug**
When overriding an asynchronous method, Pyright doesn't preface the generated return expression with the required `await` statement

When overriding an asynchronous parent method, await isn't added to the generated return expression.

**Expected**
```py
class Parent:
  async def foo(self):
    ...

class Child(Parent):
  async def foo(self):
    return await super().foo()
```
**Actual**
```py
class Parent:
  async def foo(self):
    ...

class Child(Parent):
  async def foo(self):
    return super().foo()
```

**VS Code extension or command-line**
Command line (coc-pyright)

**Additional context**
Originally opened as an issue on coc-pyright (https://github.com/fannheyward/coc-pyright/issues/442) but was told it was an issue with Pyright itself.

**Comment by erictraut** [comment]
Thanks for the suggestion. This will be addressed in the next version of pyright.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.131, which I just published. It will also be included in the next release of pylance.
